packages/algorand-js/src/interpreter/interpreter.ts

```diff
@@ -18,6 +19,8 @@ export class Interpreter {
   tx: Txn;
   gtxs: Txn[];
   accounts: AccountsMap;
+  instructions: Operator[];
+  i: number;
```

Comment:
can we name it something else ? so that it is more clear why we are using this variable

Reply:
Can you suggest something? how about `instructionIndex`?

Reply:
BTW, in the future we will need to rework it - when reporting error, we need to show a line in the source file, not the instruction index.

Reply:
yes, good point.